composer.json

```diff
@@ -1,16 +1,16 @@
 {
     "name": "setono/sylius-elasticsearch-plugin",
-    "type": "sylius-plugin",
     "description": "Elasticsearch plugin for Sylius",
+    "license": "MIT",
+    "type": "sylius-plugin",
     "keywords": [
         "sylius",
         "elasticsearch",
         "search"
     ],
-    "license": "MIT",
     "require": {
-        "php": ">=7.4",
-        "friendsofsymfony/elastica-bundle": "^5.1",
+        "php": ">=7.4 || >= 8.0",
+        "friendsofsymfony/elastica-bundle": "^6.1",
```

Comment:
Doesn't this change the Elasticsearch version also?

Reply:
I did not upgrade locally my ES, and it worked fine. But I think it can now support newer version of ES. I did not test that though.